src/Symfony/Component/Yaml/Inline.php

```diff
@@ -581,6 +584,19 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
```

Comment:
What about:
```php
                    case 0 === strpos($scalar, '!php/const:'):
                        if (self::$constantSupport && defined($const = substr($scalar, 11))) {
                            return constant($const);
                        }
                        if (self::$exceptionOnInvalidType) {
                            throw new ParseException(sprintf('The "%s" YAML string could not be parsed because the %s.', $scalar, self::$constantSupport ? ' "Yaml::PARSE_CONSTANT" flag was not set' : 'corresponding PHP constant was not defined'));
                        }

                        return; 
```